R/authenticate.R

```diff
@@ -8,15 +10,12 @@
 #'   
 #' @return sets environment variable with login information
 #' @export
-bwg_auth <- function(username, password) {
+bwg_auth <- function() {
   ## should check if it exists, and not rerun the function.
-  token_exists <- exists("token", envir = parent.frame())
-  
-  if (token_exists) stop("you've already got a token")
 
-  ## check inputs
-  assertthat::assert_that(is.character(username))
-  assertthat::assert_that(is.character(password))
+  username <- getPass::getPass("USERNAME: ")
```

Comment:
@richfitz I agree, that would be more conventional. I tried it, and it makes the username prompt appear in the interactive window, and the password prompt in a pop-up. Will that be weird?